.travis.yml

```diff
@@ -1,3 +1,5 @@
+# For Travis to run this, make sure to add a Webhook in the Github project
```

Comment:
Can you clarify what "run this" means please? Run this for pending PRs? Run this for merged PRs? Something else?